content/en/store/filecoin-plus/overview/index.md

```diff
@@ -76,6 +76,11 @@ Each request is tracked as a GitHub issue in the [Fil+ Client Onboarding repo](h
 
 To apply for a Large Dataset Notary, follow the steps at [Applying for a large DataCap allocation](https://github.com/filecoin-project/filecoin-plus-large-datasets#applying-for-a-large-datacap-allocation).
 
+### Smart Contracts
+Smart contracts can acquire DataCap just like any regular client. To do so, simply enter the address of the smart contract that requires DataCap as the client address when making a request. To receive the initial allocation of 32GiB, we recommend using the auto-verifier [Verify.glif.io](Verify.glif.io).
```

Comment:
Simply enter the *f410* address

Reply:
I would also recommend the heading say "FVM Smart Contracts" so that FVM devs know what to search for

Reply:
thanks for the review will make the edits!